### ISSUE TITLE
Standalone version

### ISSUE THREAD

**Issue by quantbits** [influxdb/influxdb#2069]
You would be surprised just how many large financial companies work on the assumption that developers do not have root access to their linux boxes. We want to try to use influxdb but can't do so without moving earth and heavens to get the sysadmin look into it. And then do it again when new version will be released, and again. I never before thought of this as a problem but here we are. 

Please, we can use a standalone executable version, that can be downloaded and run without a root access.

P.S. We tried compiling but that fails due to number of missing dependencies, that require root access to install.

**Comment by corylanou** [comment]
What errors were you getting when you tried to compile?  If you follow this document: https://github.com/influxdb/influxdb/blob/master/CONTRIBUTING.md#installing-go it should walk you through what steps are needed to compile it locally.  As far as I know, these directions are still appropriate.  If not, please let me know.

**Comment by dataminelab** [comment]
Let me try again tomorrow and I will report back, thanks.

**Comment by quantbits** [comment]
Let me try again tomorrow, I will report back, thanks.

**Comment by quantbits** [comment]
Actually, it's failing already on go1.4 installation. Go from the tarball works but when installing from source:

% gvm install go1.4

The last line of compile.log:
cmd/pprof/internal/driver
/usr/bin/ld: uncrecognized option '--build-id=none'
collect2: ld returned 1 exist status

Running Red Hat Enterprise 5.10

**Comment by corylanou** [comment]
Can you try installing go without the GVM?

https://golang.org/dl/

Not sure why that doesn't work, but it's not required if you are not doing development and just want to compile the binary.

**Comment by corylanou** [comment]
Also, we do intend to distribute just the binaries as well.

**Comment by quantbits** [comment]
Thanks, that would be awesome. Is it something we can expect during the march 0.9 stable release by any chance?

**Comment by otoolep** [comment]
We package a tarball now, which if decompressed, has a standalone binary in there.

**Comment by beckettsean** [comment]
@otoolep I think we can close this as a duplicate of https://github.com/influxdb/influxdb/issues/2070. What say you?

**Comment by otoolep** [comment]
Yeah, I think we can, but we need to do one thing.

We need to update the "Downloads" page with the following link: https://s3.amazonaws.com/influxdb/influxdb_0.9.3_x86_64.tar.gz

That way it's made known to users who wish to easily pull out a standalone binary. I have opened a ticket.

https://github.com/influxdb/influxdb.com/issues/267

**Comment by ashuptdr** [comment]
can we have influx data or influx db executables packages or files like prometheus and atlas:
e.g.
https://github.com/prometheus/prometheus/releases/download/v2.5.0/prometheus-2.5.0.linux-amd64.tar.gz

https://github.com/Netflix/atlas/releases/download/v1.5.3/atlas-1.5.3-standalone.jar
